AddressTable

```diff
@@ -0,0 +1,22 @@
+
+CREATE TABLE [dbo].[Addresses](
```

Comment:
This is supposed to be exactly like my PR here: https://github.com/Rwpulley/Portfolio/pull/1/files  These tables are supposed to be in two folders CarpetCleaning/Tables.